README.md

```diff
@@ -1,2 +1,31 @@
 # Portfolio-project-set-up-and-mobile-version-skeleton
 Project: To parse 2 sections from a Figma portfolio design.
+
+# Project Name
+
+- Setup and mobile version skeleton
+
+## Built With
+
+- HTML
+- CSS/Flexbox
+- Figma
+- Linters
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@Leon-Espitia](https://github.com/Leon-Espitia)
+- Twitter: [@LenEspitia1](https://twitter.com/Lenespitia1)
+
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## Show your support
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] Please add a `license` section here at the end of your Readme file with the `MIT` open software license like the Microverse Readme template.